cypress/integration/slidingTreeDom.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+Cypress.Commands.add('hideTree', (selector) => {
+  cy.get('ul').contains(selector).click();
+
+  cy.get('ul').contains(selector).parent().should(($ul) => {
+    const ul = $ul.find('ul').is(':hidden');
```

Comment:
we expect the value of variable ul to be equal to true or false. From the variable name it's not clear that variable contains boolena. Need a better name.

Reply:
changed to isVisible/isHidden